packages/fx-core/src/common/spec-parser/manifestUpdater.ts

```diff
@@ -124,6 +134,19 @@ export async function generateCommands(
 
               const adaptiveCardPath = path.join(adaptiveCardFolder, operationId + ".json");
 
+              if (parameters.length === 0) {
+                warnings.push({
+                  type: WarningType.OperationOnlyContainsOptionalParam,
+                  content: format(
+                    ConstantString.OperationOnlyContainsOptionalParam,
+                    method.toUpperCase(),
+                    pathUrl
+                  ),
+                  data: method.toUpperCase() + " " + pathUrl,
```

Comment:
Can we have operationId here instead?

Reply:
sure

Reply:
or even both operationId and "Method + Path"

Reply:
As discussed, will use operationId